swap_meet/item.py

```diff
@@ -1,2 +1,49 @@
+import uuid
+
+# WAVE 2
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):  # default id is None
+        """
+        Initializes an Item with a unique ID.
+        If no ID is provided, a random unique ID is generated using uuid.
+
+        Written By Mariya Mokrynska
+        """
+        if id == None: 
+            self.id = uuid.uuid4().int  # unique id
+        else:
+            self.id = id
+        self.condition = condition # assign condition value
+
+    def get_category(self):
+        """
+        Returns the name of the class as a string.
+
+        Written By Mariya Mokrynska
+        """
+        return self.__class__.__name__
+
+# WAVE 3
+
+    # Returns a string representation of the Item instance
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
```

Comment:
You wrote the `get_category` method but then don't invoke it anywhere. 

Instead of hardcoding the word "Item" here, we should use the `get_category` method so that this `__str__` can dynamically return the name of __any__ class. 

```suggestion
        return f"An object of type {self.get_category()} with id {self.id}."
```

Reply:
Replaced the hardcoded "Item" with a call to get_category in __str__ to dynamically return the class name.